### PR TITLE
Dev json preview gui

### PR DESCRIPTION
**Pull request by nikhiljose1993**
- Implemented UI to preview, edit, and download the JSON generated by your script.
- Modified JSON File to be written to `preview-gui`, instead of writing to desktop

Screenshots -

image
image

### CODE REVIEW

File: preview-gui/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
```
Comment (spell1612) [inline preview-gui/.gitignore]: Try deleting this file

**Review by spell1612 — APPROVED** [review]
Looks good to me for now.

Future Improvements:
- Please implement A common style for all the buttons used
- Clean up repetitive template code

File: preview-gui/.gitignore (hunk 1 above)
Comment (nikhiljose1993) [reply preview-gui/.gitignore]: Deleted

**Comment by nikhiljose1993** [comment]
> Looks good to me for now.
> 
> Future Improvements:
> 
> * Please implement A common style for all the buttons used
> * Clean up repetitive template codes

Will do these improvements, Thanks for merging.

**Pull request merged**